packages/carbon-graphs/dev/data/dataObjects/Line/helper.js

```diff
@@ -0,0 +1,26 @@
+const createGraphValuesWithSin = (
```

Comment:
can we rename this to file to `createGraphValues`

Reply:
As discussed since we have two math equations(sin and cos) I created two different methods with sin and cos to get the values.

Reply:
the file is called `helper` which is really generic

Reply:
Changed [here](https://github.com/cerner/terra-graphs/pull/106/commits/1cb11cb5ceddcbd7821be460253072e1d4fb091d#diff-9291820bed2ddd5388c7b9232d223bb579ec51ad71caae5af9ec996a2a0f2bfaR5).